cypress/integration/actionDeletion.spec.ts

```diff
@@ -101,13 +101,15 @@ describe('Action Deletion', () => {
                 cy.get(s.common.spinner, { timeout: constants.spinner.timeout })
                     .should('not.exist')
 
-                cy.get(s.confirmCancelModal.optionRemovePlaceholder)
+                cy.get(s.actionDeleteModal.deleteType)
+                    .find('label')
```

Comment:
I had tried to avoid using the HTML sub structure to get the dom element since it can change using various properties in OF and they either didn't work or just transferred complexity from tests to app and didn't improve it. In 
 this case the HTML is coming from Office fabric instead of our App so hopefully means it won't change as frequently.

For some reason using `:nth-child(2)` wasn't working, will have to investigate if I run into this again

Reply:
After looking more closely it seems the do allow any custom attributes such as `data-testid`s on the choices.  It wasn't documented on the website but they demo it in one of the codepens.